src/style.css

```diff
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.container {
+  height: 96px;
+  width: 96px;
+  background-color: #fff;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars__star-active {
```

Comment:
Remove this class for fixing previous comments 
use [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)

Reply:
this command highlights the column at once, I don't know how to use it.